### PR TITLE
Linguist fix

### PR DESCRIPTION
**Pull request by djdanielsson**
### What does this PR do?
forcing linguist to read yml/yaml files as a language so it stops calling the repo html

### How should this be tested?
Automated

### Is there a relevant Issue open for this?
no

### Other Relevant info, PRs, etc.
none

### CODE REVIEW

**Comment by djdanielsson** [comment]
I don't understand how it failed the lint, the extra line is there you can see that in the 2nd commit...

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.yml linguist-detectable
+*.yaml linguist-detectable
+
```
Comment (Tompage1994) [inline .gitattributes]: This is one too many lines at the end. The github UI only seems to show one of them but I've checked out your code and can see you have blank line on lines 3 and 4. If you delete line 4 then it should all pass linting.

**Review by Tompage1994 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Just the one comment on how to fix the linting error

File: .gitattributes (hunk 1 above)
Comment (sean-m-sullivan) [reply .gitattributes]: Removed it, we will see how the testing goes :)

**Review by sean-m-sullivan — APPROVED** [review]
Looks good, That fixed it. Didn't even know about this.

**Pull request merged**